src/main/java/io/github/kawamuray/wasmtime/NativeLibraryLoader.java

```diff
@@ -57,21 +60,42 @@ private static boolean tryLoadFromLibraryPath() {
     }
 
     private static String libraryPath() throws IOException {
-        String libName = "lib" + NATIVE_LIBRARY_NAME;
-        String version = libVersion();
         Platform platform = detectPlatform();
+        String version = libVersion();
         String ext = platform.ext;
-        String fileName = libName + '_' + version + '_' + platform.classifier;
+        String fileName = platform.prefix + NATIVE_LIBRARY_NAME + '_' + version + '_' + platform.classifier;
         Path tempFile = Files.createTempFile(fileName, ext);
         try (InputStream in = NativeLibraryLoader.class.getResourceAsStream('/' + fileName + ext)) {
-            Files.copy(in, tempFile, StandardCopyOption.REPLACE_EXISTING);
+            //Maybe it will be null if it's called from junit tests in some IDEs
+            if(in == null){
```

Comment:
I think we should not workaround test failures by modifying production code.
What was the problem you faced when you ran a test from IDE? In my environment (Intellij IDEA), I could run tests from IDE without any problems.
Can you make it to pass the test by configuring `java.library.path` at your IDE configuration or by adding
`systemProperty "java.library.path", "/path/to/jni-libs"` at `tests {}` configuration section of build.gradle?

Reply:
Yes you are right.  Passing the test by configuring java.library.path at Intellij IDEA's settings does help.